### PR TITLE
display opening times for pharmacies

### PR DESCRIPTION
**Pull request by markysoft**
## Checklist

- [ ] An [ADR](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions)
- [ ] New and/or updated tests
- [ ] Changes log in `CHANGELOG`

### CODE REVIEW

File: test/unit/lib/utils/formatOpeningTimes.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  expect(result.openingTimes).to.be.an('array');
+  expect(result.openingTimes.length).to.equal(2);
+  expect(result.openingTimes[0].opens).to.equal(opens1);
+  expect(result.openingTimes[0].closes).to.equal(closes1);
+  expect(result.openingTimes[1].opens).to.equal(opens2);
+  expect(result.openingTimes[1].closes).to.equal(closes2);
+}
+
+describe('formatOpeningTimes', () => {
+  it('should restrucure opening times as array, and format time', () => {
```
Comment (st3v3nhunt) [inline test/unit/lib/utils/formatOpeningTimes.js]: Typo `restrucure => restructure`

File: test/unit/lib/utils/formatTimeString.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    });
+  });
+
+  describe('displaying minutes', () => {
+    it('should display minutes if time is past the hour', () => {
+      const result = formatTimeString('13:30');
+      expect(result).to.equal('1:30pm');
+    });
+
+    it('should no display minutes if time is on the the hour', () => {
```
Comment (st3v3nhunt) [inline test/unit/lib/utils/formatTimeString.js]: Typo `no => not` and duplicate `the`

File: app/lib/utils/formatTimeString.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+const moment = require('moment');
+
+function formatTimeString(timeString) {
+  if (timeString === '23:59' || timeString === '00:00') {
+    return 'midnight';
+  }
+  const time = moment();
```
Comment (st3v3nhunt) [inline app/lib/utils/formatTimeString.js]: This block could be condensed a little and rewritten something like i.e. the moment can be created with the time initially and I think the comparison of `time.minutes() === 0` could just be `minutes === 0`.
```js
const [hours, minutes] = timeString.split(':').map(Number);
const time = moment('${hours}:${minutes}', 'HH:mm');
const formatString = minutes === 0 ? 'ha' : 'h:mma';
return time.format(formatString);
```

Comment (markysoft) [reply app/lib/utils/formatTimeString.js]: Good tip, I think I can improve on that too.

File: app/lib/utils/formatTimeString.js [hunk 4]
```diff
@@ -4,10 +4,7 @@ function formatTimeString(timeString) {
   if (timeString === '23:59' || timeString === '00:00') {
     return 'midnight';
   }
-  const time = moment();
-  const [hours, minutes] = timeString.split(':').map(Number);
-  time.minute(minutes);
-  time.hour(hours);
+  const time = moment(timeString, 'HH:mm');
```
Comment (st3v3nhunt) [inline app/lib/utils/formatTimeString.js]: Nice! I totally missed how simple it could be 🤦‍♂️

**Pull request merged**